readable.js

```diff
@@ -1,6 +1,9 @@
-var Stream = require('stream'); // hack to fix a circular dependency issue when used with browserify
+(function (){
+  try {
+    exports.Stream = require('st' + 'ream'); // hack to fix a circular dependency issue when used with browserify
```

Comment:
it will fail in the browser if streams are not already in the bundle which it won't be by default because browserify won't recognize this as requiring the stream module during the static analysis phase but *will* during the actual execution. 

Basically the only reason we are requiring streams is that so if somebody goes `myStream instance of require('stream').Readable` it will be true. which they can't do if they havn't required streams